components/TextFields/src/ColorThemer/MDCTextFieldColorThemer.h

```diff
@@ -33,7 +35,7 @@
  @param colorScheme The color scheme to apply.
  @param textInputController A MDCTextInputController instance to apply a color scheme.
  */
-+ (void)applyColorScheme:(nonnull id<MDCColorScheme>)colorScheme
++ (void)applyColorScheme:(nonnull id<MDCColorScheming>)colorScheme
```

Comment:
As discussed offline, we unfortunately can't change these APIs without it being an instant breaking change for all downstream clients.

Reply:
Done.